src/app/util/af-event.cpp

```diff
@@ -74,8 +74,8 @@ EMBER_AF_GENERATED_EVENT_CODE
 #endif // EMBER_AF_GENERATED_EVENT_CODE
 
 #if defined(EMBER_AF_GENERATED_EVENT_CONTEXT)
-uint16_t emAfAppEventContextLength        = EMBER_AF_EVENT_CONTEXT_LENGTH;
-EmberAfEventContext emAfAppEventContext[] = { EMBER_AF_GENERATED_EVENT_CONTEXT };
+uint16_t emAfAppEventContextLength                                     = EMBER_AF_EVENT_CONTEXT_LENGTH;
+EmberAfEventContext emAfAppEventContext[EMBER_AF_EVENT_CONTEXT_LENGTH] = { EMBER_AF_GENERATED_EVENT_CONTEXT };
```

Comment:
This doesn't look right.

What I would expect is that `EMBER_AF_EVENT_CONTEXT_LENGTH` matches the actual length of `EMBER_AF_GENERATED_EVENT_CONTEXT`.  It looks like people copy/pasted "4" for the former while creating different values for the latter.  `examples/all-clusters-app/all-clusters-common/gen/af-gen-event.h` has the right thing (matching values) while the other `af-gen-event.h` files do not.

So we should change the values of `EMBER_AF_EVENT_CONTEXT_LENGTH` as needed to reflect reality....

Reply:
"It looks like people copy/pasted "4" for the former while creating different values for the latter."

That's more or less what happened. Since this file is still not generated I have copy/paste it from the `all-clusters-app` and remove some code from it.
I have then copy/pasted this issue into: `src/darwin/Framework/CHIP/gen/af-gen-event.h` too...